Document config flow for decora_wifi (#18615)
home-assistant/home-assistant.github.io: source/_integrations/decora_wifi.markdown

- [DW6HD1-BZ](https://www.leviton.com/en/products/dw6hd-1bz) (Decora Smart Wi-Fi 600W Dimmer)
- [DW15S-1BZ](https://www.leviton.com/en/products/dw15s-1bz) (Decora Smart Wi-Fi 15A Switch)
- [DW15P-1BW](https://www.leviton.com/en/products/dw15p-1bw) (Decora Smart Wi-Fi Plug-in Outlet)
- [DW4SF-1BW](https://www.leviton.com/en/products/dw4sf-1bw) (Decora Smart Wi-Fi Fan Speed Controller)

> ## Setup

Reviewer: ```suggestion
## Prerequisites
```

> You will need your MyLeviton login information (username, which is usually your email address, and password) to use this module.

Reviewer: ```suggestion
You will need your MyLeviton login information (username, which is usually your email address, and password) to use this integration.
```

Reviewer: I wonder, is this set/prerequisites section really needed?

Author: Possibly not. I was following the format used by the doc page for Blink; it looks like other integrations don't put a heading over this portion of the doc.

> ## Configuration
> 
> The preferred method for setting this integration up is by using the configuration flow. Go to the integrations page in your configuration and click on new integration -> Leviton Decora Wifi. Enter your credentials in the dialog and the integration will then set up. Once Home Assistant authenticates access, the `decora-wifi` integration will create the following platforms:
> 
> - A `light` for each device linked to your MyLeviton account.
> 
> The integration reads the capabilities of each device (dimming, etc.) from the API, so devices with dimming or speed control capabilities will provide those controls in the Home assitant UI.

Reviewer: ```suggestion

{% include integrations/config_flow.md %}

Once Home Assistant authenticates access, the Leviton Decora Wi-Fi integration will create the following:

- A `light` for each device linked to your MyLeviton account.

The integration reads the capabilities of each device (dimming, etc.) from the API, so devices with dimming or speed control capabilities will provide those controls in the Home Assitant UI.
```
